python/lsst/obs/hsc/hscMapper.py

```diff
@@ -209,19 +209,23 @@ def _flipChipsLR(exp, wcs, dataId, dims=None):
         if exp:
             exp.setMaskedImage(afwMath.flipImage(exp.getMaskedImage(), flipLR, flipTB))
         if wcs:
-            wcs.flipImage(flipLR, flipTB, exp.getDimensions() if dims is None else dims)
+            ampDimensions = exp.getDimensions() if dims is None else dims
+            ampCenter = afwGeom.Point2D(ampDimensions / 2.0)
```

Comment:
I've been moving away from this in Python because it is no longer wanted in C++ (clang-format always adds spaces) and it makes going back and forth more difficult. However I am still trying to remember to use it in function arguments, and this is a good example of that.

Reply:
Surely, so long as it's our Python coding standard, you may not move away from it regardless of what `clang-format` does.